auth/src/main/AndroidManifest.xml

```diff
@@ -30,6 +30,23 @@
             android:theme="@style/FirebaseUI.Transparent"/>
 
         <activity
+            android:name=".provider.GitHubLoginHolder"
+            android:label=""
+            android:launchMode="singleInstance"
+            android:theme="@style/FirebaseUI.Transparent">
+
+            <intent-filter>
+                <action android:name="android.intent.action.VIEW"/>
+
+                <category android:name="android.intent.category.DEFAULT"/>
+                <category android:name="android.intent.category.BROWSABLE"/>
+
+                <data android:scheme="foobar"/>
```

Comment:
This is the second major issue. I couldn't figure out how to listen for the Google handle: `https://fir-ui-playground.firebaseapp.com/__/auth/handler`. I need some way for GitHub to redirect directly to our app and it wasn't working with the Google handler so I had to make up my own scheme. @samtstern @amandle Do you guys know how to do this with the Googler handler or know someone at Google who does?